atlasdb-perf/src/main/java/com/palantir/atlasdb/performance/benchmarks/TransactionGetRowsColumnRangeBenchmarks.java

```diff
@@ -42,6 +47,48 @@
     @Warmup(time = 16, timeUnit = TimeUnit.SECONDS)
     @Measurement(time = 160, timeUnit = TimeUnit.SECONDS)
     public Object getAllColumnsSingleBigRow(VeryWideRowTable table, Blackhole blackhole) {
+        return getAllRowsAndAssert(table, blackhole,
+                count -> Preconditions.checkState(count == table.getNumCols(),
+                        "Should be %s columns, but was: %s", table.getNumCols(), count));
+    }
+
+    @Benchmark
+    @Threads(1)
+    @Warmup(time = 16, timeUnit = TimeUnit.SECONDS)
+    @Measurement(time = 160, timeUnit = TimeUnit.SECONDS)
+    public Object getAllColumnsModeratelyWideRow(
+            ModeratelyWideRowTable table,
+            Blackhole blackhole) {
+        return getAllRowsAndAssert(table, blackhole,
+                count -> Preconditions.checkState(count == table.getNumCols(),
+                        "Should be %s columns, but was: %s", table.getNumCols(), count));
+    }
+
+    @Benchmark
+    @Threads(1)
+    @Warmup(time = 16, timeUnit = TimeUnit.SECONDS)
+    @Measurement(time = 160, timeUnit = TimeUnit.SECONDS)
+    public Object getAllColumnsModeratelyWideRowWithSomeUncommitted(
+            CleanModeratelyWideRowTable table,
+            Blackhole blackhole) {
+        return getAllRowsAndAssert(table, blackhole,
+                count -> Preconditions.checkState(count == table.getNumReadableCols(),
+                        "Should be %s columns, but was: %s", table.getNumReadableCols(), count));
+    }
+
+    @Benchmark
+    @Threads(1)
+    @Warmup(time = 16, timeUnit = TimeUnit.SECONDS)
+    @Measurement(time = 160, timeUnit = TimeUnit.SECONDS)
+    public Object getAllColumnsModeratelyWideRowWithManyUncommitted(
+            DirtyModeratelyWideRowTable table,
+            Blackhole blackhole) {
+        return getAllRowsAndAssert(table, blackhole,
```

Comment:
You don't need to return anything here as you're already consuming all the columns with the blackhole, no?